src/themes/dark.scss

```diff
@@ -176,8 +225,8 @@
 
   --wxc-options-list-item--selected--color:  #{$wxc-blue-40};
```

Comment:
wxc-options-list--item--selected--color

there are also in the scss of optionslist some other vars:
--wxc-options-list--normal--background
--wxc-options-list-item--focused--background
--wxc-options-list-item--active--background
--wxc-options-list-item--focus--box-shadow

they should be grouped (and focused ---> focus)